code/modules/cargo/bounties/engineering.dm

```diff
@@ -14,10 +14,10 @@
 		return FALSE
 	return T.air_contents.gases[gas_type][MOLES] >= moles_required
 
-/datum/bounty/item/engineering/gas/nitryl_tank
-	name = "Full Tank of Nitryl"
-	description = "The non-human staff of Station 88 has been volunteered to test performance enhancing drugs. Ship them a tank full of Nitryl so they can get started."
-	gas_type = /datum/gas/nitryl
+//datum/bounty/item/engineering/gas/nitryl_tank
```

Comment:
Y tho

Reply:
Really hard to make, no one really ever fills the request.